README.md

```diff
@@ -1,2 +1,17 @@
 # gtsam-project-python
 Project template using GTSAM + python wrapping
+
+
+INSTALL
+=======
```

Comment:
Using `# INSTALL` is preferred rather than putting the `=` underneath. Makes for easier editing.